internal/schema-registry/command_kek.go

```diff
@@ -42,7 +42,7 @@ type kekOut struct {
 	Doc       string `human:"Doc,omitempty" serialized:"doc,omitempty"`
 	IsShared  bool   `human:"Is Shared,omitempty" serialized:"is_shared,omitempty"`
 	Timestamp int64  `human:"Timestamp,omitempty" serialized:"timestamp,omitempty"`
-	Deleted   bool   `human:"Deleted,omitempty" serialized:"deleted,omitempty"`
+	Deleted   bool   `human:"Is Deleted,omitempty" serialized:"is_deleted,omitempty"`
```

Comment:
Take a look at other commands for how we format fields starting with "Is"
```suggestion
	IsDeleted bool   `human:"Deleted,omitempty" serialized:"is_deleted,omitempty"`
```